.circleci/config.yml

```diff
@@ -0,0 +1,34 @@
+version: 2
+jobs:
+  build:
+    docker:
+      - image: circleci/ruby:2.3.1
```

Comment:
Something to think about: Gems like this we might want to [test against multiple Ruby versions](https://jtway.co/run-tests-against-multiple-ruby-versions-on-circleci-60e1e523e245).